backend/clubs/views.py

```diff
@@ -1276,6 +1277,49 @@ def upload_file(self, request, *args, **kwargs):
 
         return file_upload_endpoint_helper(request, code=club.code)
 
+    @action(detail=True, methods=["get"])
+    def history(self, request, *args, **kwargs):
+        """
+        Return a simplified approval history for the club.
+        ---
+        responses:
+            "200":
+                content:
+                    application/json:
+                        schema:
+                            type: array
+                            items:
+                                type: object
+                                properties:
+                                    approved:
+                                        type: boolean
+                                    approved_on:
+                                        type: string
+                                        format: date-time
+                                    approved_by:
+                                        type: string
+                                        description: >
+                                            The full name of the user who approved
+                                            the club.
+                                    approved_comment:
+                                        type: string
+                                    history_date:
+                                        type: string
+                                        format: date-time
+                                        description: >
+                                            The time in which the specific version
+                                            of the club was saved at.
+        ---
+        """
+        club = self.get_object()
+        return Response(
+            ApprovalHistorySerializer(
+                club.history.order_by("history_date"),
```

Comment:
Should this be ordered in reverse, so that recent approvals are first?

Reply:
I think Julian's handling it on the frontend, but would prefer to handle it here.